seqr/views/utils/export_utils.py

```diff
@@ -77,10 +86,16 @@ def export_multiple_files(files, zip_filename, file_format='csv', add_header_pre
                 content += '\n'.join([
                     DELIMITERS[file_format].join([row.get(key) or blank_value for key in header]) for row in rows
                 ])
-                if not isinstance(content, unicode):
-                    content = unicode(content, errors='ignore')
-                zip_file.writestr('{}.{}'.format(filename.encode('utf-8'), file_format), content)
+                if isinstance(content, str):
+                    content = content.encode('utf-8')
+                content = str(content, 'ascii', errors='ignore') # Strip unicode chars in the content
+                zip_file.writestr('{}.{}'.format(filename, file_format), content)
         temp_file.seek(0)
         response = HttpResponse(temp_file, content_type='application/zip')
-        response['Content-Disposition'] = 'attachment; filename="{}.zip"'.format(zip_filename.encode('utf-8'))
+        try:
+            zip_filename.encode('ascii')
+            response['Content-Disposition'] = 'attachment; filename="{}.zip"'.format(zip_filename)
+        except UnicodeEncodeError:
+            response['Content-Disposition'] = "attachment; filename*=utf-8''{}".format(
+                quote('{}.zip'.format(zip_filename).encode('utf-8')))
```

Comment:
Python 3 can work without `.encode('utf-8')` because the `quote()` will do encoding automatically. But, in Python 2, explicitly encoding is required.

Reply:
This is all way more complicated. We should just do the same encoding for the filename that we do for the content itself and strip out all unicodes before we get here (so similar to
```
if isinstance(content, str):
       content = content.encode('utf-8')
content = str(content, 'ascii', errors='ignore')

Reply:
I would rather keep the existing behaviors unicodes in filenames than simply trip them out. So, I changed a little bit for Python 3 compatible by moving the `.encode('utf-8')` to the end of the string.